### ISSUE TITLE
struct support

### ISSUE THREAD

**Issue by WorksOfLiam** [sitemule/ILEusion#12]
Be able to pass `struct` as a type in a program/function call.

```
Dcl-Ds MyStruct Qualified Template;
  CharField Char(20);
  intField Int(10);
End-Ds;

Dcl-Pi FAK103;
  TheStruct LikeDS(MyStruct);
End-Pi;
```

```
{
	"object": "FAK103",
	"library": "ILEUSION",
	"args": 
	[
		{
			"type": "struct",
			"value": [
				{
					"type": "char",
					"length": 20,
					"value": "Hello world"
				},
				{
					"type": "int",
					"length": 10,
					"value": 234254
				}
			]
		}
	]
}
```

**Comment by WorksOfLiam** [comment]
https://www.ibm.com/support/knowledgecenter/SSPSQF_9.0.0/com.ibm.etools.iseries.langref.doc/rzan5mst41.htm#xxcvth

**Comment by WorksOfLiam** [comment]
Implemented in 37b8ee746c4eb07479e20d1bd34679af7df436eb and 3537daab58375699b63e6848eefa25174474a502.
